src/main/java/com/softserve/kickscooter/kickscootervehicle/management/converter/ScooterToInfoDTOConvertor.java

```diff
@@ -0,0 +1,22 @@
+package com.softserve.kickscooter.kickscootervehicle.management.converter;
+
+
+import com.softserve.kickscooter.kickscootervehicle.management.dao.Scooter;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterInfoDto;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ScooterToInfoDTOConvertor implements Converter<Scooter, ScooterInfoDto> {
+    @Override
+    public ScooterInfoDto convert(Scooter scooter) {
```

Comment:
Suffixes like Info, Manager, Data, etc. are too general, so aren't recommend unless it means something different besides an entity itself.